docs/manual-setup.md

```diff
@@ -0,0 +1,87 @@
+# Manual Setup Instructions for Smart Load Balancing
+
+## :gear: Setup Instructions
+
+This document outlines the steps to manually set up smart load balancing for OpenAI endpoints using Azure API Management.
+
+### Step 1: Provision Azure API Management Instance
+- Provision an [Azure API Management instance](https://learn.microsoft.com/en-us/azure/api-management/get-started-create-service-instance).
+- Ensure enabling `Managed Identity` during provisioning.
```

Comment:
I think this means "Ensure you enable `Managed Identity` during provisioning."
Are you referring to checking the "Status" checkbox on the "Managed identity" tab?

![image](https://github.com/Azure-Samples/openai-apim-lb/assets/74415662/1f7ef60d-e487-43eb-ae66-09c31feb14c2)

(by the way, that's some terrible UX. "Enabled" would be much better than simply "Status".)